account/src/repositories/accountRepository.js

```diff
@@ -12,7 +12,6 @@ export async function saveAccount(account) {
     await client.connect();
     const usersCollection = await getUsersCollection(client);
     await usersCollection.insertOne(account);
-    await client.close();
```

Comment:
i think if you remove the close it's possible to open multiple connections at the same time

Reply:
adjusted.